src/proto/loaded_image/mod.rs

```diff
@@ -59,4 +59,9 @@ impl LoadedImage {
             ucs2::decode(ucs2_slice, buffer).map_err(|_| LoadOptionsError::BufferTooSmall)?;
         core::str::from_utf8(&buffer[0..length]).map_err(|_| LoadOptionsError::NotValidUtf8)
     }
+
+    /// Get (image_base, image_size) for the loaded image.
```

Comment:
I think it's more intuitive if instead of a tuple we'd say something like: `Returns the base address and size in bytes of the loaded image`